app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -392,13 +412,17 @@ public Study updateStudy(Study study, boolean isAdminUpdate) {
             study.setEmailVerificationEnabled(originalStudy.isEmailVerificationEnabled());
             study.setExternalIdValidationEnabled(originalStudy.isExternalIdValidationEnabled());
             study.setExternalIdRequiredOnSignup(originalStudy.isExternalIdRequiredOnSignup());
+            study.setEmailSignInEnabled(originalStudy.isEmailSignInEnabled());
```

Comment:
Added all these flags to the test that compares admin and developer ability to update (developer can't, admin can).